packages/hooks/src/useResizeObserver.ts

```diff
@@ -0,0 +1,41 @@
+import { useCallback, useLayoutEffect, useRef, useState } from "react";
+import ResizeObserver from "resize-observer-polyfill";
+
+export function useResizeObserver() {
+  const [entry, setEntry] = useState<ResizeObserverEntry>();
+  const [node, setNode] = useState() as any;
```

Comment:
I feel that this `any` type needs to be resolved. This is where I got hung up in the spike was trying to figure out the types for this.

Reply:
Ok, I'll take a look here.

Reply:
@seanhealy I could use your input on this question about `any` as well. I looked but don't know how to resolve it.

Reply:
@seanhealy and I went through this and he figured out a solution. That's been committed in the last few commits.